README.md

```diff
@@ -45,33 +76,52 @@ aws lambda invoke --function-name data-portal-api-dev-migrate output.json
 aws lambda invoke --function-name data-portal-api-dev-lims_scheduled_update_processor output.json
 ```
 
+> If lambda timeout error then try again. Lambda needs warm-up time and LIMS rows are growing.
+
 ## Serverless
 
-- You can `serverless` invoke or deploy from local. But favour over CodeBuild pipeline for deploying into production environment.
-- Serverless deploy target only to AWS. Therefore need to setup AWS environment specific variables as follows:
+> :hand: Above :point_up: sections are good enough for up and running Portal backend for local development purpose. You can take on Serverless and Deployment sections below for, when you want to extend some aspect of Portal backend REST API or lambda functions and, deploying of those features.
+
+- First, install `serverless` CLI and its plugins dependencies:
+```
+node --version
+v12.18.2
+
+npm i -g yarn
+yarn global add serverless
```

Comment:
We shall have a trello card to deprecate serverless in favor of CDK... #mentalnote.